packages/@okta/vuepress-site/docs/reference/api/event-types/index.md

```diff
@@ -13,7 +13,7 @@ Event types are the primary method of categorization within the Okta eventing pl
 
 ## Catalog
 
-The following is a full listing of event types used in the [System Log API](/docs/reference/api/system-log/) with associated description and related metadata. For migration purposes it also includes a mapping to the equivalent event type in the legacy [Events API](/docs/reference/api/events/).
+The following is a full listing of event types used in the [System Log API](/docs/reference/api/system-log/) with associated description and related metadata. For migration purposes it also includes a mapping to the equivalent event type in the legacy Events API.
 The relationship between System Log API and Events API event types is generally one-to-many. Note that there are currently some System Log API event types which do not have an Events API equivalent.
```

Comment:
@brianduffield-okta - We should just strike this whole paragraph about relationship between system log and Events API.